src/adapter/src/coord/sequencer.rs

```diff
@@ -1716,10 +1717,8 @@ impl<S: Append + 'static> Coordinator<S> {
         self.catalog_transact(Some(session), ops, |_| Ok(()))
             .await?;
         for (instance_id, replicas) in instance_replica_drop_sets {
-            for (replica_id, replica) in replicas {
-                self.drop_replica(instance_id, replica_id, replica.config)
-                    .await
-                    .unwrap();
+            for replica_id in replicas {
+                self.drop_replica(instance_id, replica_id).await?;
```

Comment:
Why does this get a `?` but we still use `.unwrap()` for `add_replica_to_instance`?